src/main/kotlin/pl/krakow/riichi/chombot/Main.kt

```diff
@@ -20,16 +22,17 @@ fun main() {
     client.eventDispatcher.on(ReadyEvent::class.java)
         .subscribe { ready -> println("Logged in as " + ready.self.username) }
 
-    val commandMap = mapOf(
-        "chombo" to ChomboCommand()
+    val commandMap: Map<String, Command> = mapOf(
+        "chombo" to ChomboCommand(),
+        "_inflation" to AkagiInflationRate()
```

Comment:
It is not a real command (but behaves *almost* like one), hence the underscore. Maybe it'd be nice to separate commands and non-commands at some time in the future.

Reply:
That's okay for now. We'll either stick to that, or move to another solution that will probably emerge when we'll have more commands implemented.